stripe/src/main/java/com/stripe/android/view/CheckoutViewModel.kt

```diff
@@ -0,0 +1,60 @@
+package com.stripe.android.view
+
+import android.app.Application
+import androidx.lifecycle.AndroidViewModel
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.ViewModelProvider
+import androidx.lifecycle.liveData
+import com.stripe.android.ApiRequest
+import com.stripe.android.PaymentConfiguration
+import com.stripe.android.StripeApiRepository
+import com.stripe.android.StripeRepository
+import com.stripe.android.model.ListPaymentMethodsParams
+import com.stripe.android.model.PaymentMethod
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.Dispatchers
+
+internal class CheckoutViewModel internal constructor(
+    application: Application,
+    private val publishableKey: String,
+    private val stripeAccountId: String?,
+    private val stripeRepository: StripeRepository,
+    private val workDispatcher: CoroutineDispatcher = Dispatchers.IO
+) : AndroidViewModel(application) {
+
+    fun getPaymentMethods(
+        customerId: String,
+        ephemeralKey: String,
+        stripeAccountId: String? = this.stripeAccountId
+    ) = liveData(workDispatcher) {
```

Comment:
can this use the default view model dispatcher?

Reply:
I could be missing something but I get strict mode violations when I do this

Reply:
discussed offline - we should make getPaymentMethods (and probably other functions in StripeApiRepsoitory) suspend functions, but that will a larger refactor in a different PR